tdrs-backend/tdpservice/users/models.py

```diff
@@ -101,6 +102,9 @@ class User(AbstractUser):
             "Users in an approved state are allowed access."
         ),
     )
+
+    access_requested = models.DateTimeField(default=datetime.datetime(9999, 1, 1, 0, 0, 0))
```

Comment:
could we name it `access_requested_date` for clarity, and remove the `default` in favor of `null=True`? I think having a null date is clearer than an arbitrary default